.env

```diff
@@ -3,6 +3,6 @@ APP_DEBUG=true
 APP_KEY=wUZvVVKJyHFGDB9qK_Lop4QE1vwb4bYU
 
 DB_HOST=127.0.0.1
-DB_NAME=yii2_starter
+DB_NAME=yii2starter
 DB_USER=root
-DB_PASS=12345
\ No newline at end of file
+DB_PASS=root
```

Comment:
please do not update .env for now. I will move it as example later, once rbac extension and settings will be moved to separate extensions